cdci_data_analysis/analysis/drupal_helper.py

```diff
@@ -42,6 +42,22 @@ def analyze_drupal_output(drupal_output, operation_performed=None):
         return drupal_output.json()
 
 
+def execute_drupal_request(url, params, method='get', headers=None, files=None):
+    try:
+        if method == 'get':
+            return requests.get(url,
+                                params={**params},
+                                headers=headers)
+        elif method == 'post':
+            return requests.post(url,
+                                 data={**params},
+                                 files=files,
+                                 headers=headers
+                                 )
+    except ConnectionError as e:
+        logger.warning("connection to the drupal instance failed %s", e)
```

Comment:
what will the user  see? This way we get a warning and that's it?

User should get a notice that `gallery is inaccessible, we are investigating`
And we should get a sentry.

Actually we do now, so it's not so bad.
Just that user gets 500.

Reply:
It was not complete ye, indeed this is still a draft.

But thanks for the suggestion.

So, you actually got a sentry?

Reply:
I got a sentry. That was ok.

But user got 500. It's fine, but not ideal.